packages/sw-lib/demo/sw.js

```diff
@@ -0,0 +1,37 @@
+/* global goog */
+
+importScripts('./sw-lib.v0.0.13.min.js');
+importScripts('./manifest.123456.js');
+
+goog.swlib.cacheRevisionedAssets(self.__file_manifest);
+
+// Specific URL defined with a string.
+goog.swlib.router.registerRoute('/example/', goog.swlib.staleWhileRevalidate());
+
+// A regular expression to catch a range of strings.
+// These only need to match part of the URL.
+goog.swlib.router.registerRoute(
+  /\/images\/(.*\/)?.*\.(png|jpg|jpeg|gif)/,
+  goog.swlib.cacheFirst()
+);
+
+// Use express style routes to capture URLs
+goog.swlib.router.registerRoute('/styles/:filename', goog.swlib.cacheFirst());
```

Comment:
Showing off URL capture when the handler doesn't make use of it could be confusing.

I never really came up with a good use case for URL capture, though...

Probably more common and worth illustrating would be using `/styles/(.*)` as a wildcard?